### PR TITLE
CLI now creates default docker files

### LINKED ISSUE #4: Generate Docker file
As a user, I want the Dockerfile that is generated from hammer-cli to be populated with the industry standard node.js configurations.

- [ ] Populate Dockerfile with the standard node.js configurations.

### PR DESCRIPTION
**Pull request by KrashLeviathan**
closes #4: Generate Docker file

The change generates a Dockerfile and .dockerignore when the "Docker" option is selected from the container options.

**REVIEWER**: @jackcmeyer

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -11,10 +11,52 @@ import fs from 'fs';
 import ciChoices from '../ci-choices.json';
 import containerizationChoices from '../containerization-choices.json';
 
+function initDocker(config) {
+  console.log('Creating Dockerfile and .dockerignore...');
+
+  const dockerFileContents = `# Use the official Node runtime as a parent image
```
Comment (nischayv) [inline src/cli.js]: You should move this to a different file. Just create a new folder that can contain these files - Dockerfile, travis, etc etc

File: src/cli.js [hunk 2]
```diff
@@ -11,10 +11,52 @@ import fs from 'fs';
 import ciChoices from '../ci-choices.json';
 import containerizationChoices from '../containerization-choices.json';
 
+function initDocker(config) {
```
Comment (jackcmeyer) [inline src/cli.js]: move this to a new file

File: src/cli.js [hunk 3]
```diff
@@ -11,10 +11,52 @@ import fs from 'fs';
...
+
+# Bundle app source
+COPY . .
+
+# Make port 8080 available to the world outside this container
+EXPOSE 8080
+
+# Run "npm start" when the container launches
+CMD ["npm", "start"]
+`;
```
Comment (jackcmeyer) [inline src/cli.js]: can we make this a template file?

Comment (KrashLeviathan) [reply src/cli.js]: What's the node way of doing that?

Comment (jackcmeyer) [reply src/cli.js]: by template file, I mean just its own file

File: src/cli.js (hunk 2 above)
Comment (KrashLeviathan) [reply src/cli.js]: If we move the string literal to a template file, I don't think moving this method to a new file would be necessary.

Comment (jackcmeyer) [reply src/cli.js]: just for sanity sake we should move it to a new file, eventually we could have more tools, and could take advantage of ES6 classes and OOO

**Comment by jackcmeyer** [comment]
Can we add a test for the writeFiles() function?

**Pull request merged**